package.json

```diff
@@ -11,8 +11,7 @@
   ],
   "main": "dist/index.js",
   "scripts": {
-    "test": "vitest",
-    "build": "tsc"
+    "test": "tap src/**/*.spec.ts"
```

Comment:
nitpick: keeping it as vitest is fine, we are planning to port tap to vitest at some point

Reply:
If we're pulling it in I want the tests to run as "normal part of all the unit tests"